recipes/esmvalcore/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "ESMValCore" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b53df55c6a4d5d803ee30960bd0d7b4c2f6ea68c5f6532c8c17990d50fe5837
+
+build:
+  # Increment the build number when building a new conda package of the same
+  # esmvalcore version, reset to 0 when building a new version.
+  number: 0
+  # This could be noarch, except for the following missing binary dependencies
+  # - esmpy
```

Comment:
I don't understand. esmpy is available on [conda-forge](https://anaconda.org/conda-forge/esmpy)?

Reply:
Hey @dopplershift, thanks for taking an interest! Unfortunately, esmpy is not build for windows on conda-forge (see [here](https://github.com/conda-forge/esmpy-feedstock/blob/b57e4029702db612907b2b2f0cfa1b82409a89e2/recipe/meta.yaml#L23)). I did consider working on changing that, but I think this could/should go in before.

Reply:
So it's fine to make this package `noarch: python` and depend on `esmpy` even if the dependency isn't available everywhere. Anyone trying to install this package on Windows would just get an error about being unable to resolve dependencies.

Reply:
The problem is that the tests will fail because the CI machinery here will fail to install the dependencies. Ergo no green checkmarks, ergo no merge. Or maybe I am misunderstanding the conda-forge policy? Though the discussion in conda-forge/esmpy-feedstock#44 seems to suggest that my understanding is correct.

Reply:
Seems fine.